packages/d3fc-axis/src/axisLabelRotate.js

```diff
@@ -0,0 +1,100 @@
+import { rebindAll, exclude } from '@d3fc/d3fc-rebind';
+
+export default (adaptee) => {
+
+    let labelRotate = 'auto';
+    let decorate = () => { };
+
+    const measureLabels = s => {
+        const labels = adaptee.scale().domain();
+
+        // Use a test element to measure the text in the axis SVG container
+        const tester = s.append('text');
+        const labelHeight = tester.text('Ty').node().getBBox().height;
```

Comment:
In the label offset adapter I [measure the height](https://github.com/d3fc/d3fc/pull/1263/files#diff-2c22a2488cd6e13efc5f0d9087a94ec7R14) - would that work here too?

Reply:
I've made the two measuring functions the same now, and extracted the shared code.